src/components/pages/service/support.tsx

```diff
@@ -36,16 +36,30 @@ const ContentStyled = styled.div`
   ${TextStyles.textR}
 `;
 
-export const Intro = ({ illustration, headline, children }: any) => {
+const Kicker = styled.span`
+  ${TextStyles.toplineS}
+  display: block;
+  margin-bottom: 12px;
+  color: #3e61ee;
+  ${up('md')} {
+    ${TextStyles.toplineR}
+    margin-bottom: 16px;
+  }
+`;
+
+export const Intro = ({ illustration, headline, children, kicker }: any) => {
   const HeadlineStyledWithAnchor = WithAnchorHOC(HeadlineStyled);
   return (
-    <IntroLayout>
-      <IllustrationStyled size={IllustrationSize.LARGE} show={illustration} />
-      <div>
-        <HeadlineStyledWithAnchor>{headline}</HeadlineStyledWithAnchor>
-        <ContentStyled>{children}</ContentStyled>
-      </div>
-    </IntroLayout>
+    <>
+      {kicker && <Kicker>{kicker}</Kicker>}
+      <IntroLayout>
+        <IllustrationStyled size={IllustrationSize.LARGE} show={illustration} />
+        <div>
+          <HeadlineStyledWithAnchor>{headline}</HeadlineStyledWithAnchor>
+          <ContentStyled>{children}</ContentStyled>
+        </div>
```

Comment:
I would place the kicker in this `div` for two reasons: 
* Kicker, headline and content belong together semantically and should therefore also be together in the code
* Currently the design looks broken on mobile because of the illustration. I know that the illustration will be hidden on mobile. But if we show it again in the future, it would work that way directly.

Reply:
good point, let's put it in div